practitioner-service/src/main/resources/bootstrap.yml

```diff
@@ -3,4 +3,20 @@ spring:
     name: practitioner-service
   cloud:
     config:
-      uri: http://placeholder:1234
+      # If cannot connect to config server, don't start the application
+      fail-fast: true
```

Comment:
does this mean it wont try again? need to make sure our cd pipeline deploys the config server first

Reply:
We can remove it, but in any case the CD pipeline in this architecture should deploy: 1.Discovery 2.Config 3.All the rest